examples/multipole/run_multipole.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 ImpactX contributors
+# Authors: Ryan Sandberg, Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+# -*- coding: utf-8 -*-
+
+import amrex
+from impactx import ImpactX, distribution, elements
+
+sim = ImpactX()
+
+# set numerical parameters and IO control
+sim.set_particle_shape(2)  # B-spline order
+sim.set_space_charge(False)
+#sim.set_diagnostics(False)  # benchmarking
+sim.set_slice_step_diagnostics(True)
+
+# domain decomposition & space charge mesh
+sim.init_grids()
+
+# load a 2 GeV electron beam with an initial
+# unnormalized rms emittance of  nm
+energy_MeV = 2.0e3  # reference energy
+charge_C = 1.e-9  # used without space charge
+mass_MeV = 0.510998950
+qm_qeeV = -1.0e-6/mass_MeV  # charge/mass
+npart = 10000  # number of macro particles
+
+distr = distribution.Waterbag(
+    sigmaX = 4.0e-3,
+    sigmaY = 4.0e-3,
+    sigmaT = 1.0e-3,
+    sigmaPx = 3.0e-4,
+    sigmaPy = 3.0e-4,
+    sigmaPt = 2.0e-3,
+    muxpx = 0.0,
+    muypy = 0.0,
+    mutpt = 0.0)
```

Comment:
Zero should be the default for these :)
```suggestion
    sigmaPt = 2.0e-3)
```